src/vm/docs/mod.rs

```diff
@@ -336,7 +336,7 @@ If a value did not exist for this key in the data map, the function returns `fal
 const FETCH_CONTRACT_API: SpecialAPI = SpecialAPI {
     name: "fetch-contract-entry",
     input_type: "ContractName, MapName, Tuple",
-    output_type: "Optional<Tuple>",
+    output_type: "An optional Tuple",
```

Comment:
This should be `Optional(Tuple)`

Reply:
@kantai Is `Optional` a keyword or something?